controller/auth/user.go

```diff
@@ -175,7 +194,13 @@ func CreateJWT(email string, host string) ([]byte, error) {
 
 func Load(r *mux.Router, db *sqlx.DB) {
 	u := &userController{db}
-	r.HandleFunc("", u.Create).Methods("PUT")
-	r.HandleFunc("/login", u.Login).Methods("POST")
-	r.Use(auth.RequireNoAuthentication())
+	createUserSubR := r.PathPrefix("").Subrouter()
```

Comment:
This seems overly broad (all requests to "/" will be routed into this sub-router)?

You can make it more specific (and fewer lines of code) with:
```
createUserSubR := r.PathPrefix("").Methods("PUT").Subrouter().Use(authRequireNoAuthentication());
createUserSubR.HandlerFunc(u.Create);
```

Same applies below for
- [ ] `loginSubR`
- [ ] `renewSubR`